rskj-core/src/main/java/co/rsk/peg/bitcoin/UtxoUtils.java

```diff
@@ -0,0 +1,94 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.bitcoinj.core.VarInt;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.spongycastle.util.encoders.Hex;
+
+public final class UtxoUtils {
+
+    private UtxoUtils() {
+    }
+
+    /**
+     * Decode a {@code byte[]} of encoded outpoint values.
+     *
+     * @param encodedOutpointValues
+     * @return {@code List<Coin>} the list of outpoint values decoded in the same ordered they were
```

Comment:
`the list of outpoint values decoded preserving the order of the entry` sounds better to me. More aligned with what you put below

Reply:
i think saying "same order they were before encoded" is confusing, you are just decoding not encoding and decoding

Reply:
Done